src/map/script.cpp

```diff
@@ -10492,23 +10492,23 @@ BUILDIN_FUNC(makepet)
 BUILDIN_FUNC(getexp)
 {
 	TBL_PC* sd;
-	int base=0,job=0;
+	int64 base=0,job=0;
```

Comment:
Guess these could be defined below with the parse or remove the assignment here.